decred/decred/dcr/account.py

```diff
@@ -1865,7 +1865,7 @@ def addressSignal(self, addr, txid):
             # signal the balance update
             self.signals.balance(self.calcBalance())
 
-    def sendToAddress(self, value, address, feeRate=None):
+    def sendToAddress(self, value, address):
```

Comment:
This is fine for now. We may want to add this back as a way to override the default fee rate at some point.